kokoro/linux/build.sh

```diff
@@ -84,6 +84,8 @@ set -e
 # Build the release packages.
 mkdir $BUILD_ROOT/out
 $SRC/kokoro/linux/package.sh $BUILD_ROOT/out
+ARTIFACT_DIR=$BUILD_ROOT/out/dist
+mkdir -p $ARTIFACT_DIR
```

Comment:
`out/dist` is deleted and re-created by the `package.sh` script, so really no reason to create it here. Better to put these things somewhere else in `out`, since `dist` is meant for the distributable part of the build, not for side-artifacts.

Reply:
I was using out/dist because of the Kokoro config:

```
    # b/151297033: save trace of sample on swiftshader, for debug purposes                                                                                                   
    regex: "out/dist/*.gfxtrace"
    strip_prefix: "out/dist"
```

But I get your point, I just updated to have the gfxtrace artifact outsite of out/dist/